tests/integration/cmd_namespace_test.go

```diff
@@ -0,0 +1,27 @@
+package integration
+
+import (
+	. "github.com/onsi/ginkgo"
+
+	"github.com/redhat-developer/odo/tests/helper"
+)
+
+var _ = Describe("create/delete/list/get/set namespace tests", func() {
+	var commonVar helper.CommonVar
+
+	BeforeEach(func() {
+		commonVar = helper.CommonBeforeEach()
+	})
+	AfterEach(func() {
+		helper.CommonAfterEach(commonVar)
+	})
+	It("should successfully create the namespace", func() {
+		helper.Cmd("odo", "create", "namespace", "my-namespace").ShouldPass()
```

Comment:
You will need to delete these namespaces after the tests, or they will stay in the cluster and make future tests fail